tools/fast/tests/retrieval_test.go

```diff
@@ -119,6 +119,9 @@ func TestRetrievalLocalNetwork(t *testing.T) {
 	RunRetrievalTest(ctx, t, miner, client, sectorSize)
 }
 
+// Current issue:
+// miner state LastPoSt is nil during deal finalization, so verifyPieceInclusion will never succeed
+
```

Comment:
What is the fix for this? Do we need to be able to wait for a PoSt for the sector containing the deal piece to show up on chain?

Reply:
We are going to not rely on `LastPoSt` when checking PIPs.  We are simply going to ensure that the miner is in a valid proving period and that the commitment is on-chain before we verify the PIP.

Reply:
So does this comment go away?